tests/e2e/dao/suite.go

```diff
@@ -268,7 +268,7 @@ func (s *E2ETestSuite) TestPoPResult() {
 		challenges[i].Initiator = val.Address.String()
 		challenges[i].Challenger = aliceAddr.String()
 		challenges[i].Challengee = bobAddr.String()
-		challenges[i].Success = true
+		challenges[i].Success = blockHeight%2 == 0
```

Comment:
Can you put a comment here? Why is challenge only successful on even block heights?

Reply:
done